packages/size-limit/get-config.js

```diff
@@ -1,13 +1,18 @@
 import bytes from 'bytes-iec'
 import { globby } from 'globby'
+import _jiti from 'jiti'
 import { lilconfig } from 'lilconfig'
 import { createRequire } from 'node:module'
 import { dirname, isAbsolute, join, relative } from 'node:path'
-
+import { fileURLToPath } from 'node:url'
 import { SizeLimitError } from './size-limit-error.js'
 
 const require = createRequire(import.meta.url)
 
+const __filename = fileURLToPath(import.meta.url)
```

Comment:
I am against creating `__filename` magic constant in ESM files.

Let’s just use `jiti = _jiti(fileURLToPath(import.meta.url), { interopDefault: true })`